src/components/datepicker/js/calendarMonthBody.js

```diff
@@ -38,6 +42,12 @@
             monthBodyCtrl.generateContent();
           }
         });
+        
+        scope.$watch(function () { return calendarCtrl.minDate + calendarCtrl.maxDate; }, function (range, oldRange) {
```

Comment:
Can we avoid this watcher? maybe when `calendarCtrl.minDate` or `calendarCtrl.maxDate` are being changed we should than activate `monthBodyCtrl.generateContent();` ?

Reply:
@mvdiener is this something that you can address?

Reply:
Sure I will take a look and see if I can figure that out. I've also noticed the yearBodyCtrl will need something similar.